equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java

```diff
@@ -716,4 +717,15 @@ default String convertCaseType(Crm14DetailsModel model) {
         }
         return "Unknown";
     }
+    @Named("convertPartnerInvolved")
+    default String convertPartnerInvolved(String s) {
+        if  (s == null || s.isEmpty())
+            return null;
+        return switch (Integer.parseInt(s)) {
```

Comment:
Can we put case 1, 2 and 3 values in a const string? `private final static string`

Reply:
done